edk2toollib/database/tables/instanced_inf_table.py

```diff
@@ -164,7 +164,12 @@ def _build_inf_table(self, dscp: DscP):
         """
         inf_entries = []
         for (inf, scope, overrides) in dscp.Components:
-            logging.debug(f"Parsing Component: [{inf}]")
+            # Ignore components built with an architecture that is not in TARGET_ARCH
+            arch = scope.split(".")[-1].upper()
```

Comment:
For my education, why the switch from forward index like with the removed `"ARCH": scope.split(".")[0].upper(),` to backward index?

Reply:
This was actually a mistake on my part. Thank you for catching this. It has been switched back to 0